docs/superpowers/plans/2026-08-15-10-onboarding-playbook.md

```
@@ -0,0 +1,2044 @@
+       AIProviderName.YOUR_PROVIDER,
+       async (modelName?, _providerName?, sdk?) => {
+         const { YourProvider } = await import("../providers/yourProvider.js");
+         return new YourProvider(modelName, sdk as NeuroLink | undefined);
+       },
+       YourModels.DEFAULT,
+       ["alias1", "alias2"],
+     );
+     ```
+  ````
+
+  5. If multimodal: add vision capabilities to `ProviderImageAdapter.VISION_CAPABILITIES`
```

markdownlint-cli2 (0.23.2)

[warning] Ordered list item prefix
Expected: 1; Actual: 5; Style: 1/2/3

(MD029, ol-prefix)